src/NServiceBus.Core/Hosting/Helpers/AssemblyScannerConfiguration.cs

```diff
@@ -26,6 +26,11 @@ public class AssemblyScannerConfiguration
         /// </summary>
         public bool ScanAssembliesInNestedDirectories { get; set; }
 
+        /// <summary>
+        /// Defines an additional path for assembly scanning.
+        /// </summary>
+        public string AdditionalAssemblyScanningPath { get; set; }
```

Comment:
We did not bother to verify if the provided custom value is a directory or not. Anything that is incorrect value will fail at with an appropriate exception:  https://github.com/Particular/NServiceBus/blob/e8874d3e6c7a361f823ec47fda76ef3fc50b62f6/src/NServiceBus.Core/Hosting/Helpers/AssemblyScanner.cs#L309